### ISSUE TITLE
Allow creators to enter alt text for cover images

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#3729]
As an author, I want to be able to enter personalized alt text for my cover image, so that I can better help more readers (including blind and visually impaired people) to experience my book. 

## Notes
Currently, the Pressbooks book info page allows users to upload any image they like to serve as the book cover. At present, the book's cover image is displayed on the webbook's home page with the alt text "Cover image for BOOK TITLE" (this has been the case since https://github.com/pressbooks/pressbooks-book/pull/182 was implemented 6 years ago). 

A web & user experience librarian from Wilfrid Laurier University noted that they were unable to add customized alt text for their cover image and requested the ability to do so, noting that book cover design is an important part of a book's presentation to readers and sharing a recording of a blind user's testimony to publishers in a recent "Accessible Publishing Summit":

> the few sites and pages that I do see that have alt texts for their book covers, it's really cool because I think it really helps me understand what's in the cover. And if other sighted people are talking about it, I can talk about it with them as well. 

([source](https://www.youtube.com/live/GTanHtsOKjc?t=845s))

**Comment by MichSpieler** [comment]
This has been requested again by a network manager. [Link](https://pressbooks.zendesk.com/agent/tickets/24044) (for Support only)
